extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -160,17 +161,19 @@ public static function onAuthPluginAutoCreate( User $user ) {
 	 * Hook function to store registration data
 	 * Saves user data into the cu_changes table
 	 *
-	 * @param $user User
+	 * @param $user    User
 	 * @param $byEmail bool
+	 *
 	 * @return bool
 	 */
 	public static function onAddNewAccount( User $user, $byEmail ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$byEmail". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)